packages/clay-shared/src/useTransitionHeight.ts

```diff
@@ -40,16 +40,42 @@ export function useTransitionHeight(
 ) {
 	const [transitioning, setTransitioning] = React.useState<boolean>(false);
 
+	let transitionTimer: any;
+
+	const fakeTransitionEnd = () => {
+		const {transitionDelay, transitionDuration} = window.getComputedStyle(
+			contentRef.current
+		);
+		const totalDuration =
+			(parseFloat(transitionDelay) +
+				parseFloat(transitionDuration) +
+				0.01) *
```

Comment:
@pat270 this magic value here, is it just for the timeout not to run before calling a `transitionEnd`?

Reply:
Yes, this is a buffer. If `transitionEnd` fires the `setTimeout` is cleared in `handleTransitionEnd`.

Reply:
Maybe we may want to add this value to a constant to add more context. What do you think?

Reply:
I like the idea of adding at least a comment above this value explaining why its needed. I dont know if its needed to be a constant at the top of the file since we don't use it in multiple places, but the comment with context would be great.

Reply:
Yeah, this is another good option.